release-notes.md

```diff
@@ -123,7 +124,7 @@ There should be no fees set up.**
 
 ### Known issues
 
-None listed
+| Get auth rule txn breaks the indy-node service if plugins are on  | [ST-566](https://sovrin.atlassian.net/browse/ST-566)  |
```

Comment:
I would put it into the very top of 1.1.41 release notes. Also please make sure that the table is correctly created. (It's shown as | .. | ... |)

Reply:
Done.